src/index.html

```diff
@@ -11,12 +11,73 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="nav-logo">
+        <a href="homepage">
+          <img
+            src="images\logo.png"
+            alt="MOYO-logo"
+          />
+        </a>
+      </div>
+      <nav class="nav">
+        <div class="nav-links">
+          <ul>
+            <li class="nav-item is-active apple-active">
```

Comment:
Remove unnecessary spaces between sibling elements. There should be an empty line between multiline sibling elements, but not spaces.